charts/qdrant/Chart.yaml

```diff
@@ -13,7 +13,7 @@ maintainers:
     url: https://github.com/qdrant
 icon: https://qdrant.github.io/qdrant-helm/logo_with_text.svg
 type: application
-version: 1.15.0
```

Comment:
Please do not bump the version in the PR. If merged, we will automatically release this with the next Qdrant patch release to keep the version numbers aligned.

Reply:
Got it - will fix